src/posts/introducing_lucky_0.15.cr

```diff
@@ -0,0 +1,32 @@
+class IntroducingTheLuckyBlog < BasePost
+  title "Introducing Lucky 0.15 and a brand new website to go with it."
+  slug "introducing-lucky-website"
+
+  def published_at
+    Time.utc(year: 2020, month: 5, day: 16)
+  end
+
+  def summary
+    <<-TEXT
+    Lucky v0.15 is a big new release. Brand new class based UI components,
+    lots of bug fixes, and support for Crystal 0.29. We are also releasing a
+    brand new website to go with it.
+    TEXT
+  end
+
+  def content
+    <<-MD
+    Lucky 0.15.0 has been in the making for a few months and has lots of new
+    goodies, bug fixes, and support for the newest version of Crystal (0.29.0)
+
+    We're also releasing a brand new website along with this new version of Lucky.
+    The website is mobile friendly, has a blog (this one!) and is written using Lucky.
+
+    Enjoy!
+
+    ## New class based components
+
+    ## New built-in security features
```

Comment:
I decided to rename the file so the diff was easier to read. Here is where I plan to highlight new stuff

Thoughts on what should be included?

Reply:
Maybe you can mention it similar to how rails does it like:

### Lucky framework
fallback routing
security header modules
some bug fixes

### Avram
sequences

### Router
### Flow
### Documentation
a new websites...